clients/web/src/lib/components/organisms/scrims/modals/JoinScrimModal.svelte

```diff
@@ -1,42 +1,74 @@
 <script lang="ts">
+    import {slide} from "svelte/transition";
     import {joinScrimMutation} from "$lib/api";
-    import {Modal} from "$lib/components";
+    import {Modal, toasts} from "$lib/components";
 
     export let visible = true;
     export let scrimId: string;
 
+    let groupCode: string;
+    let joiningAsGroup: boolean = false;
+    let joining: boolean = false;
 
     async function joinSolo() {
+        joiningAsGroup = false;
```

Comment:
```suggestion
        joiningWithExistingGroup = false;
```
same, trying to make life easy by suggesting it at all points of use